eclipse-collections/src/main/java/org/eclipse/collections/impl/bag/immutable/AbstractImmutableBag.java

```diff
@@ -240,4 +242,12 @@ public ImmutableShortBag collectShort(ShortFunction<? super T> shortFunction)
         }
         return result.toImmutable();
     }
+
+    @Override
+    public ImmutableSet<T> selectUnique()
+    {
+        ImmutableBag<T> selected = this.selectByOccurrences(occurrences -> occurrences == 1);
```

Comment:
Similar to review comments of primitive side, this can be made efficient by using forEachWithOccurrences().